Node/src/TeamsMessage.ts

```diff
@@ -41,19 +41,75 @@ export enum MentionTextLocation {
   AppendText
 }
 
+/**
+ * @class
+ * At mention entity in message.
+ *
+ * @member {string} [type] at mention type, its value is always mention.
+ *
+ * @member {object} [mentioned] mentioned object with id, type and text value.
+ *
+ * @member {string} [text] text value to display in the message
+ *
+ */
+export class MentionEntity {
+  type: string;
+  mentioned: any;
+  text: string;
+}
+
+export class UserMention extends MentionEntity {
```

Comment:
All these classes and members (including the constructor) need doc comments.

Reply:
okay